### ISSUE TITLE
Building a new telepad controls generates new bluespace crystals.

### ISSUE THREAD

**Issue by Somekindofpony** [VOREStation/VOREStation#1422]
If you build a telepad in science, it starts with new crystals, take it apart and put it back together? More crystals.

**Comment by OrbisAnima** [comment]
either we edit the amount the telepad starts with, or we edit the requirements in crystals for the new one

**Comment by Arokha** [comment]
The crystals are supposed to be required to BUILD it. That might have been accidentally omitted.

**Comment by OrbisAnima** [comment]
last time I created one it required one, maybe its producing more?

**Comment by OrbisAnima** [comment]
I was unable to replicate it with artificial crystals, armed one, tried to finish it without a cristal but was unable. Then I dissasembled one and it only ejected one cristal.

**Comment by Virgo113** [comment]
Steps to reproduce are basically, build a telepad controls console. The computer one. Finish building it, do nothing more and simply disconnect the screen with a screwdriver.
Marvel as blusepace crystals magically materialize

**Comment by OrbisAnima** [comment]
oh... I tested it with the machine telepad :P

**Comment by SpadesNeil** [comment]
Was this ever fixed?

**Comment by OrbisAnima** [comment]
Nope, bug still there, it is also weird, becouse the console never gets a crystal in its construction, but the telepad does. The weird thing is, the console does uses the crystals, so either you need to add bluespace crystals to the construction of the console, and I suggest, remove them from the telepad, or dunno...

**Comment by OrbisAnima** [comment]
(you could also just hardcode the console into having max telecrystals and then just not give them telecrystals)

**Comment by lbnesquik** [comment]
Fixed.
https://github.com/VOREStation/VOREStation/pull/3474
